main/saturn-node.js

```diff
@@ -106,16 +113,21 @@ async function start (/** @type {Context} */ ctx) {
     const readyHandler = data => {
       output += data.toString()
 
-      const webuiMatch = output.match(/^WebUI: (http.*)$/m)
-      if (webuiMatch) {
-        webUrl = webuiMatch[1]
+      const apiMatch = output.match(/^API: (http.*)$/m)
+      if (apiMatch) {
+        apiUrl = apiMatch[1]
 
         appendToChildLog('Saturn node is up and ready')
-        console.log('Saturn node is up and ready (Web URL: %s)', webUrl)
+        console.log('Saturn node is up and ready (API URL: %s)', apiUrl)
+        webUrl = apiUrl + 'webui'
```

Comment:
```suggestion
        webUrl = `${apiUrl}webui`
```

modernize all the things

Reply:
I am not convinced this is an improvement - how about micro-performance? - but I am okay with following your style.